pep-0689.rst

```diff
@@ -90,16 +94,16 @@ Changes during the Beta period
 Since the API itself can change continuously up until Beta 1 (feature freeze)
 of a minor version, major users of this API are unlikely to test
 Alpha releases and provide feedback.
-It is very difficult to determine what needs to be exposed as semi-stable.
+It is very difficult to determine what needs to be exposed as unstable.
 
-Additions to the semi-stable tier will count as *stabilization*,
+Additions to the unstable tier will count as *stabilization*,
 and will be allowed up to Release Candidate 1.
```

Comment:
This sentence now reads oddly, although it makes sense on the second reading (I think!)

A

Reply:
Yes, the terminology is not perfect -- there's something more unstable than the unstable API. 
But as discussed in the thread, it's better than “semi-stable”.